README.md

```diff
@@ -37,3 +37,7 @@ The repository has the following permanent branches:
 You can reach maintainers team @mentorship-team on [Systers Open Source Slack](http://systers.io/slack-systers-opensource/) or @systers/maintainers-mentorship-android on GitHub.
 
 We use [#mentorship-system](https://systers-opensource.slack.com/messages/CAE8QK41L/) channel on Slack to discuss this project. If you're interested in contributing to this project, join us there!
+
+## License
+
+Mentorship System is Copyright © 2007 Free Software Foundation, Inc. It is free software,and may be redistributed under the terms specified in the [LICENSE FILE](LICENSE).
```

Comment:
Instead of saying all of the copyright stuff, you can jsut say something like:
```suggestion
Mentorship System is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](LICENSE) file.
```
I am actually not sure, the best way to put it, but I think leaving the internals of the license out of the readme, and kept to the file is good.

Reply:
Thank you so much for the feedback Isabela! Super sorry for the late response. I was at a summit. I think yours is better. I'll make the changes now